app/policies/common_attribute_state_policy.rb

```diff
@@ -0,0 +1,29 @@
+class CommonAttributeStatePolicy < ApplicationPolicy
+  def index?
+    user_in_admin_group?
+  end
+
+  def show?
+    user_in_admin_group?
+  end
+
+  def create?
+    user_in_admin_group?
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    user_in_admin_group?
+  end
+  
+  def edit?
+    update?
+  end
+
+  def destroy?
```

Comment:
Please remove methods that are not used for authorization. Leave create? and new? 
If we add more methods to the controller we'll add them to the policy file too.